pkg/controller/datavolume-controller.go

```diff
@@ -1390,6 +1405,27 @@ func (r *DatavolumeReconciler) reconcileProgressUpdate(datavolume *cdiv1.DataVol
 	return reconcile.Result{RequeueAfter: 2 * time.Second}, nil
 }
 
+func (r *DatavolumeReconciler) getCsiDriverForStorageClass(storageClassName *string) (bool, error) {
```

Comment:
I think this function should be named something like StorageClassCSIDriverExists or something of that nature. We are not actually returning the CSIDriver object, we are just determining if it exists, but the name suggests we are returning it.

Reply:
Yes